File: python/request_status.py
import requests

def request_status():
    private_key = ''
    url = 'https://logistic.groupngs.com/api'
    data = {
        'action': 'requestStatus',
        'requestID': '7638bae085e...1f24cf80edff1213'
    }
    headers = {
        'Authorization': 'Bearer ' +  private_key
    }
    req = requests.post(url, data=data, headers=headers)
    req.raise_for_status()
    print(req.json())

File: typescript/requestStatus.ts
import * as axios from 'axios'

interface RequestStatus {
    action: string,
    requestID: string,
}

const requestStatus = () => {
    let privateKey = '';
    let config = {
        headers: {
            Authorization: 'Bearer ' + privateKey
        }
    }

    let data: RequestStatus = {
        action: "requestStatus",
        requestID: "7638bae085e...1f24cf80edff1213"
    }

    axios.default.post('https://logistic.groupngs.com/api/', data, config)
        .then(res => {
            console.log(res.data);
        })
        .catch(error => {
            console.error(error)
        })
}

export default requestStatus;